templates/includes/sidebar.html

```diff
@@ -1,6 +1,21 @@
 <ul class="sidebar-nav">
+  {% if  user.is_authenticated %}
+    <li><a href="{% url 'taxi:driver-detail' user.pk %}">{{ user.username }}</a></li>
+  {% else %}
+    <li><a href="{% url "taxi:login" %}">Login</a></li>
```

Comment:
Inconsistent use of quotation marks in the template tag. Consider using single quotes consistently for URL template tags, like `{% url 'taxi:login' %}`.